src/ResourceManager/Resources/Commands.Resources/StartupScripts/DefaultResourceGroup.psm1

```diff
@@ -0,0 +1,38 @@
+﻿# Set Default Resource Group for Resources cmdlets
+$dllPath = '../../../../../Package/Debug/ResourceManager/AzureResourceManager/AzureRM.Resources/Microsoft.Azure.Management.ResourceManager.dll'
```

Comment:
This would only get the dll as part of a debug build. We need this to run on a user's machine, right?  Also, seems loike we should get this list from the psd1 or psm1 file.  Also, I wonder if we should generate a static script as part of the build.

Finally, how are we testing this code.  For example, what happens if Get-AzureRmContext is null